CMakeLists.txt

```diff
@@ -256,14 +256,24 @@ else()
 
   # Enable C++11 by default if found in ROOT
   if(ROOT_HAS_CXX11 AND NOT DISABLE_CXX11)
-    message(STATUS "Enabling C++11")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+    # Check whether C++11 flag is already in
+    if (${CMAKE_CXX_FLAGS} MATCHES "-std=c\\+\\+11") 
+      message(STATUS "C++11 already enabled")
```

Comment:
Hi, can we drop the messages saying "already enabled" and make the code more compact? The following would do:

```
message(STATUS "Enabling C++14")
if (NOT ${CMAKE_CXX_FLAGS} MATCHES "-std=c\\+\\+14") 
  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
endif()
```

Eventually we are not interested in seeing whether the flag is duplicated or not, we just need to know whether AliRoot understood that we want C++11/14 enabled.